firstvoices/backend/serializers/media_serializers.py

```diff
@@ -246,6 +249,29 @@ def to_representation(self, value):
         return ImageSerializer(context=self.context).to_representation(value)
 
 
+class RelatedVideoLinksSerializer(serializers.Serializer):
+    video_link = serializers.SerializerMethodField()
+    embed_link = serializers.SerializerMethodField()
+    thumbnail = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_video_link(obj):
+        return obj
+
+    @staticmethod
+    def get_embed_link(obj):
+        backend = detect_backend(obj)
+        return backend.get_url()
```

Comment:
Just wondering, if these methods are executed with every API hit, should we cache these attributes to save on response time ? or do we need these calculated dynamically for each request ?

Reply:
Caching has been added. Thanks for the suggestion!